gc/verbose/handler_standard/VerboseHandlerOutputStandard.cpp

```diff
@@ -372,6 +394,28 @@ MM_VerboseHandlerOutputStandard::handleCompactEndInternal(MM_EnvironmentBase* en
 
 #endif /* defined(OMR_GC_MODRON_COMPACTION) */
 
+void
+MM_VerboseHandlerOutputStandard::handleConcurrentMarkEnd(J9HookInterface** hook, uintptr_t eventNum, void* eventData)
+{
+	MM_ConcurrentGlobalPhaseEndEvent *event = (MM_ConcurrentGlobalPhaseEndEvent *)eventData;
```

Comment:
With renaming of hooks, we delete the more generic type `J9HOOK_MM_PRIVATE_CONCURRENT_PHASE_END`. We have to cast the event into one specific type, local or global.